apps/perf-test/tasks/flamegraph/helpers.js

```diff
@@ -0,0 +1,17 @@
+// TODO: test regex on mac, linux and windows
+const frameworkName = new RegExp(
```

Comment:
Not sure you need the constructor call here since the param you're passing in is already a regex (not a string).